src/Squirrel.CommandLine/Windows/ReleaseCommand.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+
+namespace Squirrel.CommandLine.Windows
+{
+    public class ReleaseCommand : SigningCommand
+    {
+        public Option<Uri> BaseUrl { get; }
+        public Option<string> AddSearchPath { get; }
```

Comment:
This only has an arity of 1, but should it have an arity of 0 to many?

Reply:
This argument as well as `--mainExe` (windows only) should allow 0 to many. Additionally `--addSearchPath` probably makes more sense as a global option.

Reply:
I updated both of these to both have an arity of 0 to many (this is the default for options of arrays).

I am not sure what is meant by making `--addSearchPath` a global option. Do you mean allow it to be specified for any command?

Reply:
Yes, `--addSearchPath` should probably be available for any command. If you are providing updated/custom binaries (eg. 7z, rcedit, wix, signtool) you can put them in a folder, and add that folder to the search path list, and Squirrel will find your version first instead of the bundled version.